### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by JSI31**
Some words have been changed 

ID: 27054296

### CODE REVIEW

File: res/values-ar/strings.xml [hunk 1]
```diff
@@ -1239,7 +1239,7 @@
     <string name="Absorbed_">مُنهَمِك:</string>
     <string name="team_warning">" جعل خصمُك من الفريق سيؤدي ذلك إلى حظرك من اللعبة."</string>
     <string name="Search_Filter">تصفيّة البَحِث</string>
-    <string name="Received_">تم الاستلام:</string>
+    <string name="Received_">تم الاستلام</string>
```
Comment (Jeii20) [inline res/values-ar/strings.xml:1242]: Please return the colon here.

**Review by Jeii20 — CHANGES_REQUESTED** [review]

**Comment by Jeii20** [comment]
@Moodi86 please check this when you can

File: res/values-ar/strings.xml (hunk 1 above)
Comment (JSI31) [reply res/values-ar/strings.xml:1242]: Do you mean the sorter?:

**Review by JSI31 — COMMENTED** [review]
lD:27054296

File: res/values-ar/strings.xml (hunk 1 above)
Comment (Jeii20) [reply res/values-ar/strings.xml:1242]: Yes

**Comment by JSI31** [comment]
I will fix it

**Review by JSI31 — COMMENTED** [review]
lD:27054296

**Comment by JSI31** [comment]
@Jeii20 My friend, the file has been changed

**Pull request closed without merging**